Prox/Prox/Data/PlacesController.swift

```diff
@@ -0,0 +1,104 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import AFNetworking
+import Foundation
+import Deferred
+
+/*
+ * Delegate methods for updating places asynchronously.
+ * All methods on the delegate will be called on the main thread.
+ */
+protocol PlacesControllerDelegate: class {
+    func placeControllerWillStartFetchingPlaces(_ controller: PlacesController)
+    func placesController(_ controller: PlacesController, didReceivePlaces places: [Place])
+    func placeControllerDidFinishFetchingPlaces(_ controller: PlacesController)
+    func placesController(_ controller: PlacesController, didError error: Error)
+}
+
+private let apiSuffix = "/api/v1.0/at/%f/%f"
+
+class PlacesController {
```

Comment:
Based on a comment that @mcomella made on another patch where I was trying to copy this pattern, perhaps a different name approach would be `PlacesProvider` so that we don't get confused between ViewControllers nd Model Controllers?